res/themes/light-custom/css/_custom.scss

```diff
@@ -0,0 +1,110 @@
+//
+// --accent-color
+$accent-color: var(--accent-color);
+$button-bg-color: var(--accent-color);
+$button-link-fg-color: var(--accent-color);
+$button-primary-bg-color: var(--accent-color);
+$input-valid-border-color: var(--accent-color);
+$reaction-row-button-selected-border-color: var(--accent-color);
+$roomsublist-chevron-color: var(--accent-color);
+$tab-label-active-bg-color: var(--accent-color);
+$togglesw-on-color: var(--accent-color);
+$username-variant3-color: var(--accent-color);
+$accent-color-50pct: var(--accent-color-50pct); //still needs alpha at .5
+//
+// --timeline-background-color
+$authpage-body-bg-color: var(--timeline-background-color);
+$button-secondary-bg-color: var(--timeline-background-color);
+$field-focused-label-bg-color: var(--timeline-background-color);
+$lightbox-border-color: var(--timeline-background-color);
+$menu-bg-color: var(--timeline-background-color);
+$avatar-bg-color: var(--timeline-background-color);
+$message-action-bar-bg-color: var(--timeline-background-color);
+$primary-bg-color: var(--timeline-background-color);
+$roomtile-focused-bg-color: var(--timeline-background-color);
+$togglesw-ball-color: var(--timeline-background-color);
+$droptarget-bg-color: var(--timeline-background-color-50pct); //still needs alpha at .5
```

Comment:
fwiw if we update/use a different scss engine we can use `opacity()` instead of this naming convention.

Reply:
the css variables are evaluated at runtime, so that still wouldnt work. but yes, for other themes that could be useful.